katalog/pomerium/svc.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pomerium
+spec:
+  ports:
+    - port: 80
+      name: http
+      targetPort: 8080
```

Comment:
The POD definition exposes port 443 instead of 8080. Is it OK?

Reply:
you are right @ralgozino , although it doesn't affect the functionality, I will align it with the real port used

Reply:
this now should be ok!